backend/dal/Services/PropertyService.cs

```diff
@@ -12,33 +14,59 @@ public class PropertyService : IPropertyService
         private readonly ClaimsPrincipal _user;
         private readonly ILogger _logger;
         private readonly IPropertyRepository _propertyRepository;
+        private readonly ICoordinateTransformService _coordinateService;
 
-        public PropertyService(ClaimsPrincipal user, ILogger<PropertyService> logger, IPropertyRepository propertyRepository)
+        public PropertyService(ClaimsPrincipal user, ILogger<PropertyService> logger, IPropertyRepository propertyRepository, ICoordinateTransformService coordinateService)
         {
             _user = user;
             _logger = logger;
             _propertyRepository = propertyRepository;
+            _coordinateService = coordinateService;
         }
 
         public PimsProperty GetById(long id)
         {
             _logger.LogInformation("Getting property with id {id}", id);
             _user.ThrowIfNotAuthorized(Permissions.PropertyView);
-            return _propertyRepository.Get(id);
+
+            // return property spatial location in lat/long (4326)
+            var property = _propertyRepository.Get(id);
+            if (property?.Location != null)
+            {
+                var newCoords = _coordinateService.TransformCoordinates(SpatialReference.BC_ALBERS, SpatialReference.WGS_84, property.Location.Coordinate);
+                property.Location = GeometryHelper.CreatePoint(newCoords, SpatialReference.WGS_84);
+            }
+            return property;
         }
 
         public PimsProperty GetByPid(string pid)
         {
             _logger.LogInformation("Getting property with pid {pid}", pid);
             _user.ThrowIfNotAuthorized(Permissions.PropertyView);
-            return _propertyRepository.GetByPid(pid);
+
+            // return property spatial location in lat/long (4326)
+            var property = _propertyRepository.GetByPid(pid);
+            if (property?.Location != null)
+            {
+                var newCoords = _coordinateService.TransformCoordinates(SpatialReference.BC_ALBERS, SpatialReference.WGS_84, property.Location.Coordinate);
+                property.Location = GeometryHelper.CreatePoint(newCoords, SpatialReference.WGS_84);
```

Comment:
since this is overriding the entity information, what do you think about using a new variable instead? Not to be done now but something to think about since the location does not represent what is in the repository anymore.

Reply:
I've been thinking about where's the best place to do this conversion. We can talk more this Friday and decide on approach.